docs/run.md

```diff
@@ -0,0 +1,101 @@
+2022-01-31T11:23:22.972Z	DEBUG	applicationEventLoopRunner - processing event	{"gitopsDeplUID": "9a4a6a74-900b-4101-b9a2-4bdf6d72060a", "workspaceID": "815fa61b-0c05-437e-9f72-401e7cefecfd", "event": "[DeploymentModified] default/gitopsdeployment-sample/GitOpsDeployment, for workspace '815fa61b-0c05-437e-9f72-401e7cefecfd', gitopsdepluid: '9a4a6a74-900b-4101-b9a2-4bdf6d72060a'", "attempt": 1}
+2022-01-31T11:23:22.972Z	DEBUG	Sent work to depl event runner	{"workspaceID": "815fa61b-0c05-437e-9f72-401e7cefecfd", "gitOpsDeplID": "9a4a6a74-900b-4101-b9a2-4bdf6d72060a", "event": "[DeploymentModified] default/gitopsdeployment-sample/GitOpsDeployment, for workspace '815fa61b-0c05-437e-9f72-401e7cefecfd', gitopsdepluid: '9a4a6a74-900b-4101-b9a2-4bdf6d72060a'"}
+2022-01-31T11:23:22.983Z	DEBUG	sharedResourceEventLoop receive message: getOrCreateClusterUserByNamespaceUID	{"workspace": "815fa61b-0c05-437e-9f72-401e7cefecfd"}
+2022-01-31T11:23:22.992Z	DEBUG	workspacerEventLoopRunner_handleDeploymentModified processing event	{"gitopsDeploymentExists": true, "deplToAppMapExists": false}
+2022-01-31T11:23:22.992Z	DEBUG	sharedResourceEventLoop receive message: getOrCreateSharedResources	{"workspace": "815fa61b-0c05-437e-9f72-401e7cefecfd"}
+2022-01-31T11:23:23.002Z	INFO	STUB: handleNewGitOpsDeplEvent after getOrCreateManagedEnvironmentByNamespaceUID, skipping creating the Argo CD Cluster Secret via a new Operation.
+2022-01-31T11:23:23.191Z	INFO	Creating database operation	{"gitopsDeplUID": "9a4a6a74-900b-4101-b9a2-4bdf6d72060a", "workspaceID": "815fa61b-0c05-437e-9f72-401e7cefecfd", "operation": "operation-id: , instance-id: e7020b99-65ae-415b-b944-e7e5bc89ab19, owner: 59f42d7a-6d02-4f79-965b-44073d1ad7ed, resource: 5b5a872c-2ad8-42d2-9a4e-3282e97b8c69, resource-type: Application, "}
+2022-01-31T11:23:23.197Z	INFO	Creating K8s Operation CR	{"gitopsDeplUID": "9a4a6a74-900b-4101-b9a2-4bdf6d72060a", "workspaceID": "815fa61b-0c05-437e-9f72-401e7cefecfd", "operation": "b41de877-1524-4522-8ac6-e8952c5e8172"}
+2022-01-31T11:23:23.209Z	INFO	STUB: Not waiting for create Application operation to complete, in handleNewGitOpsDeplEvent	{"gitopsDeplUID": "9a4a6a74-900b-4101-b9a2-4bdf6d72060a", "workspaceID": "815fa61b-0c05-437e-9f72-401e7cefecfd"}
+2022-01-31T11:23:23.209Z	DEBUG	Deleting operation CR: operation-b41de877-1524-4522-8ac6-e8952c5e8172	{"gitopsDeplUID": "9a4a6a74-900b-4101-b9a2-4bdf6d72060a", "workspaceID": "815fa61b-0c05-437e-9f72-401e7cefecfd", "operation": "b41de877-1524-4522-8ac6-e8952c5e8172", "namespace": "argocd"}
+2022-01-31T11:23:23.224Z	DEBUG	applicationEventQueueLoop received work complete event	{"workspaceID": "815fa61b-0c05-437e-9f72-401e7cefecfd", "gitOpsDeplID": "9a4a6a74-900b-4101-b9a2-4bdf6d72060a", "event": "[DeploymentModified] default/gitopsdeployment-sample/GitOpsDeployment, for workspace '815fa61b-0c05-437e-9f72-401e7cefecfd', gitopsdepluid: '9a4a6a74-900b-4101-b9a2-4bdf6d72060a'"}
+```
```

Comment:
Hi,
A quick suggestion, I wanted to make on our call from here is,
Instead of pasting a log here can't we make a logs/ dir inside docs/ and paste a file as backend_log.txt. That will be easier to follow.

```
docs
└── logs
    ├── backend_logs.txt
    └── cluster_agent_logs.txt
```

Reply:
Actually I wasn't even sure if it makes sense to include the logs here. My intention was to have a place where the current status of the project is captured. I would avoid creating a file.